src/service/repo/signedMessageRepo.ts

```diff
@@ -6,24 +6,39 @@ import {
   SignedPostNotification
 } from '../../../types/repo/SignedMessageRepo/SignedPostNotificationData';
 
+type SendPayloadType = {
+  channelId: string;
+  message: string;
+  channelType: number;
+  messageType?: string;
+  replyMessageId?: string;
+};
+
 const baseUrl = {
   checkIsTargetAllowingAnonDM: 'chat/channels/check-allow-anon-dm-status',
   sendSignedMessage: '/chat/send-signed-message',
   getAllSignedChannels: '/chat/channels/signed',
   getAllSignedPostNotifications: '/feeds/feed-chat',
   getSingleSignedPostNotifications: (activityId: string) => `/feeds/feed-chat/${activityId}`,
   setChannelAsRead: '/chat/channels/read',
-  createSignedChat: '/chat/channels-signed'
+  createSignedChat: '/chat/channels-signed',
+  deleteMessage: (messageId: string) => `/chat/message/${messageId}`
 };
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `baseUrl` object has been updated to include new endpoints for deleting messages. This is a good practice as it centralizes the API endpoint URLs, making them easier to manage and update if necessary.



<!-- This is an auto-generated comment by OSS CodeRabbit -->